deployments/terraform/modules/alarm_gateway/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "project" {}
+variable "environment" {}
+variable "family" {}
+variable "application" {}
+
+variable "create" {
+  type        = bool
+  default     = true
+  description = "Defines if alarm should be created"
+}
+
+variable "datapoints_to_alarm" {
+  type        = number
+  default     = 3
+  description = "The number of datapoints that must be breaching to trigger the alarm"
+}
+
+variable "evaluation_periods" {
+  type        = number
+  default     = 3
+  description = "The number of periods over which data is compared to the specified threshold"
+}
+
+variable "success_rate_threshold" {
+  type        = number
+  default     = 95
+  description = "Required percentage of successful requests"
+}
+
+variable "period" {
+  type        = number
+  default     = 300
+  description = "The period in seconds over which the specified statistic is applied"
+}
+
+variable "api_request_path" {
+  type        = string
+  default     = null
+  description = "Api request path"
+}
+
+variable "aws_lb_application_arn_suffix" {
```

Comment:
is it a suffix or do we provide the full arn?

Reply:
according to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_metric_alarm examples we have to provide suffix
also here https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-cloudwatch-metrics.html when load balancer dimenssion is used it isnt' and arn

Reply:
okay, it's the "final portion of the load balancer ARN" :+1: